modules/atomic-widgets/prop-types/shadow-prop-type.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\PropTypes;
+
+use Elementor\Modules\AtomicWidgets\PropTypes\Base\Object_Prop_Type;
+use Elementor\Modules\AtomicWidgets\PropTypes\Primitives\String_Prop_Type;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Shadow_Prop_Type extends Object_Prop_Type {
```

Comment:
calling it "shadow" might be problematic, because text-shadow & drop-shadow don't support spread and inset

Reply:
Do you have better name?